pipelines/respondent-home-ui.yml

```diff
@@ -0,0 +1,148 @@
+---
+resources:
+
+- name: respondent-home-ui-source
+  type: git
+  source:
+    uri: https://github.com/ONSdigital/respondent-home-ui.git
+    branch: aiohttp-app
+
+- name: cf-resource-latest
+  type: cf
+  source:
+    api: ((cloudfoundry_api))
+    username: ((cloudfoundry_email))
+    password: ((cloudfoundry_password))
+    organization: rmras
+    space: concourse-latest
+    skip_cert_check: true
+
+- name: cf-resource-preprod
+  type: cf
+  source:
+    api: ((cloudfoundry_api))
+    username: ((cloudfoundry_email))
+    password: ((cloudfoundry_password))
+    organization: rmras
+    space: concourse-preprod
+    skip_cert_check: true
+
+- name: cf-resource-prod
+  type: cf
+  source:
+    api: ((cloudfoundry_api))
+    username: ((cloudfoundry_email))
+    password: ((cloudfoundry_password))
+    organization: rmras
+    space: concourse-prod
+    skip_cert_check: true
+
+jobs:
+- name: respondent-home-ui-latest-deploy
+  plan:
+  - get: respondent-home-ui-source
+    trigger: true
+  - put: push-app
+    resource: cf-resource-latest
+    params:
+      current_app_name: respondent-home-ui-latest
+      manifest: respondent-home-ui-source/manifest.yml
+      path: respondent-home-ui-source
+      environment_variables:
+        HOST: 0.0.0.0
+        LOG_LEVEL: INFO
+        ACCOUNT_SERVICE_URL: http://respondent-home-ui-latest.apps.devtest.onsclofo.uk
+        EQ_URL: https://staging-new-surveys.dev.eq.ons.digital/
```

Comment:
Can we get the default `EQ_URL` entries to point at `[host]/session?token=` [as found in runner](https://github.com/ONSdigital/eq-survey-runner/blob/755681b6f4840df7d1430ea9ad2636b0275253b9/app/views/session.py#L38) so we can append the JWT to the string without hardcoding any paths/params in RH code.